apps/native/app/src/screens/wallet/wallet.tsx

```diff
@@ -139,6 +152,12 @@ export const WalletScreen: NavigationFunctionComponent = ({ componentId }) => {
     return []
   }, [res])
 
+  const lastUpdated = licenseItems.find((item) => item.fetch.updated)?.fetch
+    .updated
+  const lastUpdatedFormatted = lastUpdated
+    ? intl.formatDate(new Date(parseInt(lastUpdated, 10)))
+    : undefined
```

Comment:
~nit: This could live in useMemo

Also is this showing the latest item updated? Aka finding the licenseItem which was last updated?